src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/CSharp/CSharpProjectedDocument.ts

```diff
@@ -39,7 +39,7 @@ export class CSharpProjectedDocument implements IProjectedDocument {
         }
 
         let content = this.content;
-        for (const edit of edits) {
+        for (const edit of edits.reverse()) {
```

Comment:
Stole this fix from @ajaybhargavb

Reply:
This fix was merged. You can rebase to get it.